examples/index.json

```diff
@@ -4263,6 +4263,92 @@
       }
     ]
   },
+  {
+    "title": "Reading data",
+    "column": 0,
+    "category": "Persistence",
+    "samples": [
+      {
+        "name": "Get all",
+        "url": "persist-get-all",
+        "verifyBuild": false,
+        "verifyOutput": false,
+        "disableVerificationReason": "Needs code generation",
+        "disablePlayground": true,
+        "isLearnByExample": false
+      },
+      {
+        "name": "Get by key",
+        "url": "persist-get-by-key",
+        "verifyBuild": false,
+        "verifyOutput": false,
+        "disableVerificationReason": "Needs code generation",
+        "disablePlayground": true,
+        "isLearnByExample": false
+      },
+      {
+        "name": "Select fields",
+        "url": "persist-select-fields",
+        "verifyBuild": false,
+        "verifyOutput": false,
+        "disableVerificationReason": "Needs code generation",
+        "disablePlayground": true,
+        "isLearnByExample": false
+      },
+      {
+        "name": "Filtering and sorting",
```

Comment:
`Filter and query`?

Reply:
Fixed in [6990cb0](https://github.com/ballerina-platform/ballerina-distribution/pull/4429/commits/6990cb01fd9efd42df057ecf60daf35b7bea7945)